org.lflang/src/org/lflang/generator/cpp/CppStandaloneCmakeGenerator.kt

```diff
@@ -133,6 +133,21 @@ class CppStandaloneCmakeGenerator(private val targetConfig: TargetConfig, privat
                 |)
                 |target_link_libraries($S{LF_MAIN_TARGET} $reactorCppTarget)
                 |
+            ${" |"..targetConfig.linkLibraries.joinToString("\n") {
+                    buildString {
+                        // Need a special case for libm on Windows
+                        if (it.trim() == "m") {
+                            append("if(NOT MSVC)\n")
+                        }
+                        append("find_library(LF_${it}_LIB $it)\n")
+                        append("target_link_libraries($S{LF_MAIN_TARGET} \"$S{LF_${it}_LIB}\")\n")
+                        if (it.trim() == "m") {
+                            append("endif()\n")
+                        }
+                    }.toString()
+                }
+            }
+                |
```

Comment:
The added code snippet really breaks with the style of the class. It would be best to factor out this code in a separate (Java?) function and call it here.

Reply:
I updated the code and removed `find_library` since it appears to be unnecessary.